index.html

```diff
@@ -3,47 +3,40 @@
-          e.preventDefault()
-          window.open('https://duckduckgo.com/?q=' +
-            document.getElementById('vo-selector').value +
-            '+voor+' +
-            document.getElementById('search-target').value)
-        })
-        document.getElementById('pizza').addEventListener('click', function (e) {
-          alert('Ter vervanging van hopeloos verouderde cookies, maakt DuckDuckVo gebruik van pizza\'s.')
-          window.open('pizza.html')
-        })
-      }
-    </script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
+       integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="style/style.css">
+    
+    <script src="scripts/search.js"></script>
+    <script src="https://kit.fontawesome.com/423e0ca86c.js"></script>
   </head>
   <body>
-    <h1>DuckDuckVo</h1>
-    <h1>Privacy voor prominente studenten.</h1>
-    <form id="search-form">
-      <select id="vo-selector">
-        <option value="vo">`Vo</option>
-        <option value="anti-vo">Anti-`Vo</option>
-      </select>
-       voor 
-      <input type="text" id="search-target">
-      <input type="submit">
-    </form>
+    <main> 
+      <div class="container"> 
+        <img src="images/logo.png">
+        <h1 id="title">DuckDuckVo</h1>
 
-    <img src="images/pizza.png" onclick="pizzaTime()" class="pizza" id="pizza"
-    alt="Ter vervanging van hopeloos verouderde cookies, maakt DuckDuckVo gebruik van pizza\'s.">
+        <div class="container"> 
+          <form class="input-group mb-3" id="search-form"> 
+            <div class="input-group-prepend"> 
+              <select class="input-group-text custom-select" id="vo-selector"> <option value="'vo" selected="">'Vo</option> <option value="anti-'vo">Anti-'Vo</option> </select> 
+              <span class="input-group-text">voor</span> 
+            </div> 
+            <input type="text" class="form-control" id="search-target" /> 
+            <div class="input-group-append"> 
+              <button class="input-group-text" type="submit"><i class="fas fa-search"></i></button> 
+            </div> 
+          </form> 
+        </div>
+
+        <h5>Privacy voor prominente studenten.</h5>
+
+        <img src="images/pizza.png" onclick="pizzaTime()" class="pizza" id="pizza"
+          alt="Ter vervanging van hopeloos verouderde cookies, maakt DuckDuckVo gebruik van pizza\'s.">
+      </div>
+    </main>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
```

Comment:
Where is this dependency used? If it's not used, we can remove it.
```suggestion
```

Reply:
This is a script that comes with Bootstrap. I'll check if it's necessary in our case.

Reply:
Deleting the line doesn't change the functionality, I'll remove it and the others.

Reply:
Great, thanks!